onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -522,6 +522,7 @@
     <string name="preferences_db_save_error">Unable to save: %1$s</string>
     <string name="preferences_db_restore_warning">This will wipe out any existing data! Go ahead?
     </string>
+    <string name="preferences_restore_loading">Restoring from storage</string>
```

Comment:
Could you please add an ellipsis here?

So:
`<string name="preferences_restore_loading">Restoring from storage…</string>`

Also, please make sure it's the unicode character for the ellipsis, not just three periods.

Thanks!